pkg/omf/functions/bundle/omf.bundle.add.fish

```diff
@@ -1,5 +1,10 @@
 function omf.bundle.add -a type name_or_url
-  set -l bundle $OMF_CONFIG/bundle
```

Comment:
Yes, please.
It's always good to have a new pair of eyes looking at the code. :)